src/com/jedk1/jedcore/ability/waterbending/Drain.java

```diff
@@ -25,29 +23,15 @@
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.util.Vector;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Random;
+import java.util.*;
```

Comment:
why?

Reply:
as I wrote before, had default settings in IntelliJ IDEA at that moment.